sentry-log/src/integration.rs

```diff
@@ -42,7 +42,10 @@ impl Integration for LogIntegration {
             log::set_max_level(filter);
         }
 
-        INIT.call_once(|| log::set_boxed_logger(Box::new(Logger::default())).unwrap());
+        INIT.call_once(|| {
+            // silently ignore errors here, as running tests will hit this condition for some reason.
+            log::set_boxed_logger(Box::new(Logger::default())).ok();
+        });
```

Comment:
So tests end up running this more than once?  I'm not convinced this is the right work around in this case

Reply:
So this was super weird. Some of my local testruns were crashing hard. The global logger is not thread safe, so I suspect thats what was causing things to fail, though not sure.